src/main/java/com/example/connectfour/model/Game.java

```diff
@@ -0,0 +1,39 @@
+package com.example.connectfour.model;
+
+public class Game {
+    private String id;
+    private int[][] board; 
+    private int currentPlayer; 
+    private String status; 
+    private Integer winner; 
+    private String player1Id;
+    private String player2Id;
+
+    public Game(String id) {
+        this.id = id;
+        this.board = new int[6][7];
+        this.currentPlayer = 1;
+        this.status = "waiting";
+    }
+
+    public String getId() { return id; }
+    public void setId(String id) { this.id = id; }
+    
+    public int[][] getBoard() { return board; }
```

Comment:
I'd suggest breaking the game, players, and board out from each other as separate classes/entities